orcid-web/src/main/webapp/static/javascript/ng1Orcid/require.js

```diff
@@ -1,12 +1,36 @@
-//npm install -g typescript
+__webpack_public_path__ = window.resourceBasePath;
+
+require("../../css/glyphicons.css");
+require("../../css/social.css");
+//  require("../../css/filetypes.css");
+// require("../../css/nova-light/theme.css");
+// require("../../css/primeicons.css");
+// require("../../css/primeng.min.css");
+require("../../twitter-bootstrap/3.3.6/css/bootstrap.min.css");
+require("../../css/orcid.new.css");
+require("../../css/academicons.css");
+// require("../../css/jquery-ui-1.10.0.custom.min.css");
+// require("../../css/noto-font.css");
+require("../../css/noto-sans-googlefonts.css");
+if (window.location.href.indexOf("/print") > 0) {
+  require("../../css/orcid-print.css");
+}
 
 function requireAll(requireContext) {
   return requireContext.keys().map(requireContext);
 }
 
-require('./app/polyfills.ts');
-require('./app/main.ts');
+require("./app/polyfills.ts");
+require("jquery");
+// require("../jqueryui/1.10.0/jquery-ui.min.js");
```

Comment:
@lizkrznarich All the commented lines are the CSS and JS I am not requiring anymore, I am keeping these lines commented to easy refer the removed files in case we found a problem.